steps/redhat.py

```diff
@@ -10,6 +10,12 @@
 rh_gpg_fingerprint = '567E 347A D004 4ADE 55BA  8A5F 199E 2F91 FD43 1D51'
 rh_gpg_fingerprint_short = '199E2F91FD431D51'
 
+def get_ipaddr(context, host):
+	ip_addr = context.remote_cmd("shell",
+				host,
+				module_args="ip a s dev eth0 | grep 'inet ' | cut -d' ' -f6 | cut -d'/' -f1")[0]['stdout']
```

Comment:
You could even just use the Ansible facts provided by the `setup` module:

```
$ ansible all -i "192.168.122.220," -u cloud-user -m setup -a "filter=ansible_default_ipv4"
192.168.122.220 | success >> {
    "ansible_facts": {
        "ansible_default_ipv4": {
            "address": "192.168.122.220",
            "alias": "eth0",
            "gateway": "192.168.122.1",
            "interface": "eth0",
            "macaddress": "52:54:00:25:80:cd",
            "mtu": 1500,
            "netmask": "255.255.255.0",
            "network": "192.168.122.0",
            "type": "ether"
        }
    },
    "changed": false
}
```

The `remote_cmd()` would look something like this:

```python
ip_addr = context.remote_cmd("setup",
                                                   host,
                                                   module_args="filter=ansible_default_ipv4")
```

And then process `ip_addr` to extract the address.

You can look through the list of facts provided by Ansible here - http://docs.ansible.com/ansible/playbooks_variables.html#information-discovered-from-systems-facts

And the reference for the `setup` module here - http://docs.ansible.com/ansible/setup_module.html